README.md

```diff
@@ -17,27 +19,23 @@ cd scratch-vm
 npm install
 ```
 
-## Setup
-For an extended setup example, check out the /playground directory, which includes a fully running VM instance.
-```js
-var VirtualMachine = require('scratch-vm');
-var vm = new VirtualMachine();
+## Development Server
+This requires Node.js to be installed.
 
-// Block events
-workspace.addChangeListener(vm.blockListener);
-var flyoutWorkspace = workspace.toolbox_.flyout_.workspace_;
-flyoutWorkspace.addChangeListener(vm.flyoutBlockListener);
+For convenience, we've included a development server with the VM. This is useful because the VM can take advantage of executing in a WebWorker, which is not permitted in a local file.
 
-// Run threads
-vm.runtime.start();
+## Running The Dev Server
```

Comment:
Pretty sure "The" shouldn't be capitalized here - "Running the Dev Server" would be better.